exercises/csharp/TriviaGame/TriviaGame/GameRunner.cs

```diff
@@ -9,36 +9,37 @@ namespace TriviaGame
 {
    public class GameRunner
     {
+        private static bool _notAWinner;
+        private static Random _randomNumberGenerator;
 
-        private static bool notAWinner;
-
-        public static void Main(String[] args)
+        public static void Main(string[] args)
         {
-            Game aGame = new Game();
+            int seed = int.Parse(args[0]);
+            _randomNumberGenerator = new Random(seed);
 
-            aGame.add("Chet");
-            aGame.add("Pat");
-            aGame.add("Sue");
+            var dice = new Dice(_randomNumberGenerator);
+            Game aGame = new Game(new Board());
 
-            Random rand = new Random(Int32.Parse(args[0]));
+            aGame.AddPlayer(new Player("Chet"));
+            aGame.AddPlayer(new Player("Pat"));
+            aGame.AddPlayer(new Player("Sue"));
 
             do
             {
+                aGame.RollDice(dice);
 
-                aGame.roll(rand.Next(5) + 1);
-
-                if (rand.Next(9) == 7)
+                if (_randomNumberGenerator.Next(9) == 7)
                 {
-                    notAWinner = aGame.wrongAnswer();
+                    _notAWinner = aGame.GiveCurrentPlayerWrongAnswer();
                 }
                 else
                 {
-                    notAWinner = aGame.wasCorrectlyAnswered();
+                    _notAWinner = aGame.wasCorrectlyAnswered();
                 }
```

Comment:
I think that with naming conventions if line 33 is called GiveCurrentPlayerWrongAnswer() this should be GiveCurrentPlayerCorrectAnswer()